packages/plugins/src/SidebarPlugin.ts

```diff
@@ -204,6 +204,11 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
             }
             if (currPage !== undefined) {
               createNavigationRefs(currPage, prevPage, nextPage);
+            } else if (currPage === undefined) {
+              console.warn(
+                '\x1b[31m',
+                `*** PAGE RETURNING UNDEFINED -  PLEASE CHECK ALL YOUR DIR'S CONTAIN INDEX PAGES. THIS MAY CAUSE ISSUES WITH SIDEBAR AND NAVIGATION ***`
+              );
```

Comment:
Might be good to make this more consistent with the style of other warnings like:

console.warn('Page returning undefined. Please ensure that all directories contain index pages. This may cause issues with sidebar and navigation.');

Reply:
yeah probably best to throw an error so it will stop and then the user can read what the problem was